gensrc/proto/cloud.proto

```diff
@@ -782,6 +782,7 @@ message AlterObjStoreInfoRequest {
     optional ObjectStoreInfoPB obj = 2;
     optional Operation op = 3;
     optional StorageVaultPB vault = 4;
+    optional bool set_as_default_storage_vault = 5;
```

Comment:
consider carrying the message back to caller of this PRC to tell if there is an existing vault has been replaced.

Reply:
But how to tell the user that the original default storage vault is replaced?